### PR TITLE
openvr: add missing dlfcn dependency

### PR DESCRIPTION
**Pull request by Kreijstal**
Look at the .pc file it has -ldl flag on --libs

### CODE REVIEW

File: mingw-w64-openvr/PKGBUILD [hunk 1]
```diff
@@ -4,13 +4,13 @@ _realname=openvr
...
 pkgver=2.2.3
-pkgrel=1
+pkgrel=2
 pkgdesc="API and SDK that allows access to VR hardware from multiple vendors (mingw-w64)"
 arch=('any')
 mingw_arch=('mingw64' 'ucrt64' 'clang64' 'clangarm64')
 url="https://github.com/ValveSoftware/openvr/"
 license=('spdx:BSD-3-Clause')
-depends=("${MINGW_PACKAGE_PREFIX}-gcc-libs")
+depends=("${MINGW_PACKAGE_PREFIX}-gcc-libs ${MINGW_PACKAGE_PREFIX}-dlfcn")
```
Comment (ognevny) [inline mingw-w64-openvr/PKGBUILD:13]:
```suggestion
depends=("${MINGW_PACKAGE_PREFIX}-gcc-libs "${MINGW_PACKAGE_PREFIX}-dlfcn")
```

**Comment by Biswa96** [comment]
> Look at the .pc file it has -ldl flag on --libs

It should be removed for native Windows platform.

**Comment by Kreijstal** [comment]
> > Look at the .pc file it has -ldl flag on --libs
> 
> It should be removed for native Windows platform.

oh

**Comment by Biswa96** [comment]
dlopen and dlsym functions are not used in Windows. See the following code snippet

https://github.com/ValveSoftware/openvr/blob/ae46a8dd0172580648c8922658a100439115d3eb/src/vrcore/sharedlibtools_public.cpp#L19

**Pull request closed without merging**